assets/js/base/context/cart-checkout/payment-methods/payment-method-data-context.js

```diff
@@ -138,13 +143,15 @@ export const PaymentMethodDataProvider = ( {
 	);
 	const { setValidationErrors } = useValidationContext();
 	const { addErrorNotice, removeNotice } = useStoreNotices();
+	const { setShippingAddress } = useShippingDataContext();
 
 	const setExpressPaymentError = ( message ) => {
-		addErrorNotice( message, {
-			context: 'wc/express-payment-area',
-			id: 'wc-express-payment-error',
-		} );
-		if ( ! message ) {
+		if ( message ) {
+			addErrorNotice( message, {
+				context: 'wc/express-payment-area',
+				id: 'wc-express-payment-error',
+			} );
+		} else if ( ! message ) {
```

Comment:
This fixes a bug where an error notice with no content could get set (when the intention was to clear the notice)!

Reply:
🤦‍♂ Of course!

Reply:
fixed in 758d6a2e